docs/getting-started-guides/windows/index.md

```diff
@@ -254,6 +227,21 @@ To start your cluster, you'll need to start both the Linux-based Kubernetes cont
 ## Starting the Linux-based Control Plane
 Use your preferred method to start Kubernetes cluster on Linux. Please note that Cluster CIDR might need to be updated.
 
+## Support for kubeadm join
+
+If your cluster has been created by [kubeadm](https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/), 
+and your networking is setup correctly using one of the methods listed above (networking is setup outside of kubeadm), you can use kubeadm to add a Windows node to your cluster. At a high level, you first have to initialize the master with kubeadm (Linux), then set up the CNI based networking (outside of kubeadm), and finally start joining Windows or Linux worker nodes to the cluster. For additional documentation and reference material, visit the kubeadm link above.
+
+The kubeadm binary can be found at [Kubernetes Releases](https://github.com/kubernetes/kubernetes/releases), inside the node binaries archive. Adding a Windows node is not any different than adding a Linux node:
+
+`kubeadm.exe join --token <token> <master-ip>:<master-port> --discovery-token-ca-cert-hash sha256:<hash>`
+
+See [joining-your-nodes](https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/#44-joining-your-nodes) for more details.
+
+## Supported Features
+
+The examples listed below assume running Windows nodes on Windows Server 1709. If you are running Windows Server 2016, the examples will need the image updated to specify `image: microsoft/windowsservercore`. This is due to the requirement for container images to match the host operating system version when using process isolation.
```

Comment:
Who knows when the `:latest` tag won't apply to Server 2016 anymore? `:ltsc2016` is more explicit. Maybe a link to https://hub.docker.com/r/microsoft/windowsservercore/ for further reading would also be useful.

Reply:
This is a much better approach. Implicit tags lead to serious problems